packages/synapse-react-client/src/utils/hooks/useFadeTransition.ts

```diff
@@ -1,28 +0,0 @@
-import { useState, useEffect } from 'react'
```

Comment:
useFadeTransition() was to support a very specific case, where a Fade was being reused (when the image file handle changed, which we tracked in "state").  This is no longer the design, and I was confused by the purpose of this hook (so I can't imagine anyone else will use it!).